src/bin/travis/test.sh

```diff
@@ -4,18 +4,32 @@ set -e
 
 SCRIPT_DIR=src/bin/travis
 
-# BUILD_TYPE is coming in from the Travis build matrix
-if [ "$BUILD_TYPE" = "centaurJes" ]; then
-    "${SCRIPT_DIR}"/testCentaurJes.sh
-elif [ "$BUILD_TYPE" = "centaurTes" ]; then
-    "${SCRIPT_DIR}"/testCentaurTes.sh
-elif [ "$BUILD_TYPE" = "centaurLocal" ]; then
-    "${SCRIPT_DIR}"/testCentaurLocal.sh
-elif [ "$BUILD_TYPE" = "sbt" ]; then
-    "${SCRIPT_DIR}"/testSbt.sh
-elif [ "$BUILD_TYPE" = "checkPublish" ]; then
-    "${SCRIPT_DIR}"/testCheckPublish.sh
-else
-    echo "Unknown BUILD_TYPE: '$BUILD_TYPE'"
-    exit 1
+if [ "$TRAVIS_EVENT_TYPE" = "push" ] || [ "$TRAVIS_EVENT_TYPE" = "pull_request" ] || [ "$TRAVIS_EVENT_TYPE" = "api" ]; then
+    # BUILD_TYPE is coming in from the Travis build matrix
```

Comment:
these bash wizards have at least a couple suggestions for ways to DRY this with `case`s or pattern matches:

https://stackoverflow.com/questions/21157435/bash-string-compare-to-multiple-correct-values